documentation/asciidoc/topics/con_xsite_replication.adoc

```diff
@@ -1,5 +1,7 @@
 [id='cross-site-replication_{context}']
-= Cross-site replication with {ispn_operator}
+= Automatically forming cross-site views with {ispn_operator}
```

Comment:
Is it really "Automatic"? You have to configure the URLS in the Infinispan CR and  the cache definitions as well as exchanging tokens.

Maybe "Managed" is a more appropriate term?